ubuntu/5.0/go1.24/dockerd-entrypoint.sh

```
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -e
+
+/usr/local/bin/dockerd \
+	--host=unix:///var/run/docker.sock \
+	--host=tcp://127.0.0.1:2375 \
+	--storage-driver=overlay2 &>/var/log/docker.log &
```

Shellcheck (0.10.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)